src/service/pubsub.ts

```diff
@@ -16,9 +17,51 @@ export default class PubSubService {
   private constructor() {
     this.initDriver();
     this.initClient();
+    this.startServer();
     this.bind(this);
   }
 
+  public startServer(): void {
+    if (process.env.RUN_HEALTH_SERVER !== 'true') return;
```

Comment:
Let's prefix this env var with PUBSUB too like all other?

```suggestion
    if (process.env.PUBSUB_HEALTH_SERVER !== 'true') return;
```